### ISSUE TITLE
attestationlist.org 

### ISSUE THREAD

**Issue by amanwithwings** [metagov/daostar#209]
We need to deploy a Token Lists (https://tokenlists.org/) style front-end to track EAS schemas related to DAOs. The UI can be effectively the same. 

@Rashmi-278, we need to figure out how listing will work. 

<img width="807" alt="Screenshot 2024-03-06 at 8 46 46 PM" src="https://github.com/metagov/daostar/assets/68982632/98b8a528-1618-4109-8047-cad04e5988e4">

**Comment by Rashmi-278** [comment]
Work in progress here: https://github.com/metagov/Attestation-List

Website live: https://attestation-list.onrender.com

**Comment by Rashmi-278** [comment]
![An Attestation standard for DAOs](https://github.com/metagov/daostar/assets/41290852/a8bd1596-8e92-4cad-b3d3-3df771a0d436)

**Comment by amanwithwings** [comment]
Thanks for the quick turnaround here, @Rashmi-278.  Posting a few questions that we need to answer: 

1. Do we have a domain for this? Let's also decide on a name. 
2. How to validate that a schemas follows the proposed standard?
3. How does one author an attestation list? 
4. What details are we going to display for each schema (https://attestation-list.onrender.com/token-list?url=tokens.1inch.eth) ?
5. What extra info do we need from the list author (logo, documentation, etc.)?

**Comment by Rashmi-278** [comment]
Aman and I discussed the aboved questions on a call on 12th March 2024

**Comment by amanwithwings** [comment]
[CONTENT]

An Attestation List for DAOs.

DAO Attestation List is a community-led initiative to improve discoverability, reputation and trust of DAO-related schemas on EAS. All schemas listed on this website are DAOIP-3 compliant. 

-> Why Attestation List? 
-> Make your own 
-> Community 

[Comments]

1. There are 4 links in the top right corner (_Community, Why lists?, Make a list, GitHub_) and 3 in the bottom left corner ( _-> Why Attestation List?, -> Make your own, -> Community_). 

We may not need all of them, we also may not need to duplicate links. 

2. I think we should remove the step in which people submit an attestation list on our Github for verification. Attesting to the registry should be enough, it is what EAS seems to be doing on their ecosystem page. 

We can contact issuers for more info if necessary (or maybe have them contact us)? Since information attested to the registry will be public, we may not be able to collect their contact info through it. 

3. Buttons like"make your own list" or "list your schemas" should lead to a form as we have here: https://daostar.org/register. It could be inspired by this: https://base.easscan.org/ecosystem/submit-project. 

4. @Rashmi-278, I remember us discussing a UI for issuer profiles (these pages: https://attestations.daostar.org/token-list?url=tokens.1inch.eth). Could you deploy this design to production? 

As of now, the registry collects information including _schema UID, schema description, network ID, issuer name, issuer description, logo, and apiDocsURI._  We can also query things like number of attestations, owner, date of creation, etc. for each schema via EAS. Obviously, there is no need to include all of this info - just the necessary bits. 

5. I was thinking "Why attestation lists" could lead to a blog post or Substack entry (or the same content hosted on this website).

**Comment by Rashmi-278** [comment]
Updates live here: https://attestations.daostar.org/

1. could you share which ones we don't need?
2. Removed, we ae now indexing attestations, creating attester profile and listing the schemas
3. Form needs to be worked on 
4. Done, https://attestations.daostar.org/schemas?id=0x88e50e06efB2B748E2B9670d2a6668237167382B
We also index and display schema details
5. Sure, please share a link
6. Yes, that would make things secure

**Comment by thelastjosh** [comment]
Stub for discussion today! @amanwithwings @frm @Rashmi-278 . 

Topics covered:
- Rationale, posting here or to the related discussion at #230 
- Questions around adoption
- Could facilitate collaboration with EAS / Bryce; EAS wants to support their BD
- We will probably not be able to dedicate BD resources from DAOstar itself; need support from EAS or an independent grant for this.

Next steps
- [ ] Aman will reach out to the existing Token List GitHub group / developers, see if there is interest or room for collaboration.

**Comment by Rashmi-278** [comment]
@amanwithwings and @Rashmi-278 Discussed broadening the scope of this project.

Waiting on @frm to discuss more

**Comment by thelastjosh** [comment]
- [ ] Add somewhere to the DAOstar front page. Maybe create a Labs subpage or a Tools subpage and include it there, along with descriptions of other side projects, experiments, and stuff to come that is not yet folded into the main registration / explore pathways? ALTERNATELY, include it as a section somewhere down in the main page.

**Comment by thelastjosh** [comment]
Still waiting to add it somewhere. @Rashmi-278 Create a Labs subpage in the documentation and add this (and some other relevant projects?) there.
